### PR TITLE
feat(onboarding): Added SplashScreen

### LINKED ISSUE #4954: Implement splash screen
### Description
Create initial splash screen as shown in new designs in [figma](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba⎜Desktop-(rolled-back-version-to-recover-a-screen%2C-ignore)?node-id=1023%3A298171)

Assumptions:
- Use Lottie animation from https://lottiefiles.com/share/xl7pjmjz
- For now 2sec timer in the UI to hide the splash screen with a TODO to hook it up to the backend when ready

splash

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #4954

### What does the PR do
- Added SplashScreen as initial screen before
onboard as per new designs.
- Also moved all onboarding related code from
main to onboarding section and switched to
import this and all other sections as
qml modules, exposing this way only
necessary files.
- Cleaned up unecessary qmldir files and
imports.

Lottie asset returns an error, used static logo in the meantime until that is fixed.

### Affected areas
app start up

### Screenshot of functionality
https://user-images.githubusercontent.com/31625338/156667122-2c100336-f68a-4905-9784-a38fef7ab197.mov
https://user-images.githubusercontent.com/31625338/156667137-2ec84053-8f6d-46d6-8188-9d38c49cc9d9.mov

### CODE REVIEW

**Review by iurimatias — CHANGES_REQUESTED** [review]
when I try to run this branch I get this error

```
QQmlApplicationEngine failed to load component
qrc:/main.qml:19:1: module "applayouts.Onboarding" is not installed
```

**Comment by alexandraB99** [comment]
> when I try to run this branch I get this error
> 
> ```
> QQmlApplicationEngine failed to load component
> qrc:/main.qml:19:1: module "applayouts.Onboarding" is not installed
> ```

@iurimatias please make sure the folder `AppLayouts` in `status-desktop/ui/app/AppLayouts` has no capital letters, it should be like: `status-desktop/ui/app/applayouts`

**Comment by noeliaSD** [comment]
> > when I try to run this branch I get this error
> > ```
> > QQmlApplicationEngine failed to load component
> > qrc:/main.qml:19:1: module "applayouts.Onboarding" is not installed
> > ```
> 
> @iurimatias please make sure the folder `AppLayouts` in `status-desktop/ui/app/AppLayouts` has no capital letters, it should be like: `status-desktop/ui/app/applayouts`

Same happend to me but yes, updating the `AppLayouts` folder to not contain capital letters solves the issue!!

Nice Splash Screen!! :)

**Review by noeliaSD — APPROVED** [review]
LGTM! I like the refactor done by creating the `OnboardingLayout` and concentrating all the related flow there!

**Comment by abhi-status** [comment]
Hey @alexandraB99 !
The Windows installable does not launch the app for some reason.
I can see it run in the background (via the Windows TaskManager)
I haven't been able to test the Linux version yet, as my Ubuntu VM is corrupted (resolving this now)
I'll update here once I am able to get through.

**Comment by abhi-status** [comment]
I've just tried the installable on Linux and I am getting the same error as Iuri & Noeila. 

I cant test this via the installables, as I cant change this casing (applayouts folder) anywhere.

Can this casing issue be resolved in the installable package ?

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -3,22 +3,22 @@ import QtQuick.Controls 2.13
 import QtQuick.Layouts 1.13
 import QtMultimedia 5.13
 import Qt.labs.qmlmodels 1.0
+import Qt.labs.platform 1.1
+import Qt.labs.settings 1.0
+
+import applayouts.Wallet 1.0
```
Comment (stefandunca) [inline ui/app/mainui/AppMain.qml:9]: It seems changing the imports from lowercase `applayouts` to camel-case `AppLayouts` will fix it

Comment (stefandunca) [reply ui/app/mainui/AppMain.qml:9]: I now see the comment that it should be the other way around.

Comment (iurimatias) [reply ui/app/mainui/AppMain.qml]: PR updated to change the imports

**Pull request merged**